aws/ecs/cloudformation-identiorca.json

```diff
@@ -155,16 +157,12 @@
         "Yes"
       ]
     },
-    "UseScopeService": {
-      "Fn::Not": [
+    "UseWeaveScopeStandalone": {
```

Comment:
That's right, but my explanation above does say that and it is wrong.
Here is what [the docs](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#d0e118119) say:
```
"Fn::If": [condition_name, value_if_true, value_if_false]
```